constant/index.js

```diff
@@ -30,7 +30,7 @@ if (IS_TESTNET) whiteList.push('localhost');
 
 export const REDIRECT_WHITE_LIST = whiteList;
 
-export const REDIRECT_NAME_WHITE_LIST = ['in-tokensale', 'in-bundle'];
+export const REDIRECT_NAME_WHITE_LIST = ['in-tokensale', 'in-bundle', 'in-settings'];
```

Comment:
`REDIRECT_NAME_WHITE_LIST` is only used in `sendVerifyEmail`, so if `in-settings` is not used in email then no need add this

Reply:
User may change their email at settings page, would like to refer them back